qiskit/transpiler/passes/synthesis/unitary_synthesis.py

```diff
@@ -109,7 +140,7 @@ class UnitarySynthesis(TransformationPass):
 
     def __init__(
         self,
-        basis_gates: List[str],
+        basis_gates: List[str] = None,
```

Comment:
what does basis_gates = None mean? docstring needs to say

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7795/commits/409409ebd81566fc3d4bf604462a42d37addeb13